custom/icds_reports/models/aggregate.py

```diff
@@ -1590,6 +1592,33 @@ class Meta(object):
     _agg_atomic = False
 
 
+class AggregateAvailingServiceForms(models.Model, AggregateMixin):
+    """ Aggregated data for the availing services
+
+    A availing services exists for each state_id
+
+    A row exists for every person case that has had a record of registration
+    submitted against it this month
+    """
+    state_id = models.CharField(max_length=10)
+    supervisor_id = models.TextField(null=True)
+    awc_id = models.TextField(null=True)
+    month = models.DateField(help_text="Will always be YYYY-MM-01")
+
+    # not the real pkey - see unique_together
+    person_case_id = models.CharField(max_length=40, primary_key=True)
+
+    is_registered = models.PositiveSmallIntegerField(blank=True, null=True, help_text="Status of the Registration")
+    registration_date = models.DateTimeField(help_text="Registration Date", null=True)
```

Comment:
In the future, this kind of column can probably be a date field since we never need to know the time of a registration